packages/protocol/contracts/tokenvault/ERC1155Vault.sol

```diff
@@ -156,8 +152,6 @@ contract ERC1155Vault is BaseNFTVault, ERC1155ReceiverUpgradeable {
         (CanonicalNFT memory ctoken,,, uint256[] memory tokenIds, uint256[] memory amounts) =
             abi.decode(message.data[4:], (CanonicalNFT, address, address, uint256[], uint256[]));
 
-        if (ctoken.addr == address(0)) revert VAULT_INVALID_TOKEN();
-
```

Comment:
Removed this check because it's not necessary? There is no similar check in `receiveToken` which depends on the same data. So either the check is needed in both or in none of them at least.

Reply:
I think it's OK to remove this check as if ctoken.addr is zero, the transfer function will revert below.